modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMMDFClaim.testcase

```diff
@@ -174,28 +248,45 @@ definition {
 				thirdInvoice = "Document_1.png");
 		}
 
-		task ("And I go back to the MDF listing page") {
-			PRMUtils.gotoPRM();
+		task ("And given that the Manager user is logged in and goes to MDF listing page") {
+			User.logoutAndLoginPG(
+				userLoginEmailAddress = "manager@liferay.com",
+				userLoginFullName = "manager");
 
-			PRMHomePage.gotoQuickLinksOption(
-				buttonLink = "mdf-requests",
-				buttonText = "MDF");
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
 		}
 
 		task ("And I go to MDF claim listing page") {
 			PRMNavigator.goToNavItem(navItem = "MDF Claim");
 		}
 
-		task ("When I export all visible MDF's") {
+		task ("Then I filter the first account assigned to the claims group and verify the information inside the file") {
+			PRMMDFClaim.filterClaimID(
+				buttonOption = "Filter",
+				checkboxName = "Company Name 1",
+				nameButton = "Apply",
+				optionDropdown = "Partner");
+
 			Button.click(button = "Export MDF Claim");
+
+			PRMMDFRequest.viewExportedMDF(
+				fileContentList = "Company Name 1",
+				fileName = "MDF Claim.csv");
+
+			Refresh();
 		}
 
-		task ("Then the information about the visible MDF's will be inside the exported file") {
-			var firstID = PRMMDFClaim.getClaimID(claimIndex = 1);
-			var secondID = PRMMDFClaim.getClaimID(claimIndex = 2);
+		task ("And I filter by the second account and check the appropriate results") {
+			PRMMDFClaim.filterClaimID(
+				buttonOption = "Filter",
+				checkboxName = "Company Name 2",
+				nameButton = "Apply",
+				optionDropdown = "Partner");
+
+			Button.click(button = "Export MDF Claim");
 
 			PRMMDFRequest.viewExportedMDF(
-				fileContentList = "${firstID},${secondID}",
+				fileContentList = "Company Name 2",
 				fileName = "MDF Claim.csv");
 		}
```

Comment:
Hi @marinaasant0s, it should be inside a separated task 'Then the information about the filtered Claims will be inside the file'